strawberryfields/api/devicespec.py

```diff
@@ -136,9 +122,9 @@ def validate_parameters(self, **parameters):
             if p not in self.gate_parameters:
                 raise ValueError(f"Parameter {p} not a valid parameter for this device")
 
-    def create_program(self, **parameters):
-        """Create a Strawberry Fields program matching the low-level layout of the
-        device.
+    def create_program(self, **parameters: complex):
```

Comment:
Is this right?
```suggestion
    def create_program(self, **parameters: complex) -> sf.Program:
```

Reply:
Yes! I must confess that I was almost certain that I added this specific return type hint, but apparently I didn't. 😕

Reply:
Will using a string resolve the circular import issue? 🤔
```suggestion
    def create_program(self, **parameters: complex) -> "sf.Program":
```

Reply:
Yes, that could work. But `import strawberryfields as sf` still needs to be added, but never used (which I find a bit ugly just for the sake of a type hint). What do you think?